public/app/features/plugins/admin/hooks/usePluginInfo.tsx

```diff
@@ -19,7 +19,8 @@ export const usePluginInfo = (plugin?: CatalogPlugin): PageInfoItem[] => {
 
   // Populate info
   const latestCompatibleVersion = getLatestCompatibleVersion(plugin.details?.versions);
-  const version = plugin.installedVersion || latestCompatibleVersion?.version;
+  const useLatestCompatibleInfo = !plugin.isInstalled;
+  const version = plugin.installedVersion || (useLatestCompatibleInfo && latestCompatibleVersion?.version);
```

Comment:
Could you replace this for a ternary condition?  `(useLatestCompatibleInfo && latestCompatibleVersion?.version);`

What is the value of `version` if `useLatestCompatibleInfo` is false and `plugin.installedVersion` is falsy?

Reply:
I am not sure what you are picturing with a ternary condition but I can replace this with an `if` to make it more readable.

Reply:
@andresmgot that works better! the current code makes it hard to know what are the outputs to the possible conditions.